packages/flutter/lib/src/animation/performance.dart

```diff
@@ -310,6 +311,62 @@ class ProxyPerformance extends PerformanceView
   double get progress => _masterPerformance != null ? _masterPerformance.progress : _progress;
 }
 
+class CurvedPerformance extends PerformanceView {
```

Comment:
Proxy is mutable. This isn't. It could subclass Proxy, but then it's doing more work than it needs to. Common parts could be factored out of both, though, if you think that's valuable?